plugins/UserAgreementPlugin/UserAgreement.qml

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 1.3
+
+import UM 1.3 as UM
+
+UM.Dialog
+{
+    id: baseDialog
+    minimumWidth: Math.floor(UM.Theme.getSize("modal_window_minimum").width * 0.75)
+    minimumHeight: Math.floor(UM.Theme.getSize("modal_window_minimum").height * 0.5)
+    width: minimumWidth
+    height: minimumHeight
+    title: "User Agreement"
+
+    TextArea
+    {
+        anchors.top: parent.top
+        width: parent.width
+        anchors.bottom: buttonRow.top
+        text: ' <center><h3>DISCLAIMER BY ULTIMAKER</h3></center>
+                <p>PLEASE READ THIS DISCLAIMER CAREFULLY.</p>
+                <p>EXCEPT WHEN OTHERWISE STATED IN WRITING, ULTIMAKER PROVIDES ANY ULTIMAKER SOFTWARE OR THIRD PARTY SOFTWARE “AS IS” WITHOUT WARRANTY OF ANY KIND. THE ENTIRE RISK AS TO THE QUALITY AND PERFORMANCE OF ULTIMAKER SOFTWARE IS WITH YOU.</p>
+                <p>UNLESS REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, IN NO EVENT WILL ULTIMAKER BE LIABLE TO YOU FOR DAMAGES, INCLUDING ANY GENERAL, SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY TO USE ANY ULTIMAKER SOFTWARE OR THIRD PARTY SOFTWARE.</p>
+                '
```

Comment:
I think it's better to read this from a text file than to hard-code it in here.

Does it need to be translated?

Reply:
Not sure if we can translate this legalese, let discuss tomorrow

Reply:
Does it need to be SHOUTED?

Reply:
Some parts of an end-user license agreement indeed has to be SHOUTED (or does it?) A good breakdown here: https://www.quora.com/Why-do-some-parts-of-legal-documents-appear-in-all-caps-e-g-EULAs-have-portions-written-in-caps-lock-Does-this-mean-something-in-a-legal-sense

Reply:
Why not only have this annoying pop up on the cura download page? or will we show the bubble each time the terms change (even if this is not the first time?)

Reply:
also, how valid are the EULA's in court?

Reply:
only valid when they're SHOUTED, apparently